py/label_microservice/models.py

```diff
@@ -0,0 +1,24 @@
+"""The models packages defines wrappers around different models."""
+
```

Comment:
Consider using [abstract classes](https://julien.danjou.info/guide-python-static-class-abstract-methods/) to define python interfaces.  Not required, but might help catch errors later